syft/pkg/rpm_metadata.go

```diff
@@ -11,7 +11,7 @@ import (
 // Packages is the legacy Berkely db based format
 // Packages.db is the "ndb" format used in SUSE
 // rpmdb.sqlite is the sqlite format used in fedora + derivates
-const RpmDBGlob = "**/var/lib/rpm/{Packages,Packages.db,rpmdb.sqlite}"
+const RpmDBGlob = "**/{var/lib,usr/share}/rpm/{Packages,Packages.db,rpmdb.sqlite}"
```

Comment:
Would you be able to add a test that has a simple rpmdb in this alternate location?

Reply:
Side note: if you'd like to also support OpenSUSE, you might also need `/var/lib/sysimage/` like in https://github.com/snyk/snyk-docker-plugin/commit/27958de3366ab6fd1b7f7faad64cce8f0dce5350